stdlib/camlinternalFormat.ml

```diff
@@ -438,11 +440,12 @@ let bprint_altint_fmt buf ign_flag iconv pad prec c =
 
 (***)
 
-(* Print the optional '+' associated to a float conversion. *)
-let bprint_fconv_flag buf fconv = match fst fconv with
+(* Print the optional '+', ' ' and/or '#' associated to a float conversion. *)
+let bprint_fconv_flag buf fconv = begin match fst fconv with
   | Float_flag_p -> buffer_add_char buf '+'
   | Float_flag_s -> buffer_add_char buf ' '
-  | Float_flag_ -> ()
+  | Float_flag_ -> () end;
+  match snd fconv with Float_CF -> buffer_add_char buf '#' | _ -> ()
```

Comment:
I would prefer to avoid the fragile pattern here, by listing all the cases instead of using `_`. This will prevent a bug if we decide to add another `#..` float format later.

Reply:
Indeed, done